scaaml/io/dataset.py

```diff
@@ -477,7 +477,7 @@ def summary(dataset_path):
 
     @staticmethod
     def inspect(dataset_path,
-                split: Literal['train', 'test', 'holdout'],
+                split: str,  # typing.Literal['train', 'test', 'holdout'],
```

Comment:
Maybe we can be a bit smarter here:
```python
# Literal was introduced in Python 3.8
try:
  from typing import Literal
  SPLIT_TYPE = Literal['train', 'test', 'holdout']
except ImportError:
  SPLIT_TYPE = str
...
def inspect(dataset_path, split: SPLIT_TYPE, ...):
```

Up to you to decide.
Pros:
- we don't need to think about code changes when we'll drop py37 support

Cons:
- might be a bit less readable
- requires to disable some lint checks I presume

Reply:
I would prefer not using this approach or at least doing this in a separate issue (I have created Issue #38 for various code improvements).

I plan to leave Literal in the code which is not run on the py3.7 computer (AES capture).